docs/Configuration.md

```diff
@@ -1415,20 +1426,16 @@ These pattern strings match against the full path. Use the `<rootDir>` string to
 
 ### `testRegex` \[string | array&lt;string&gt;]
 
-Default: `(/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$`
+Default: `[]`
 
-The pattern or patterns Jest uses to detect test files. By default it looks for `.js`, `.jsx`, `.ts` and `.tsx` files inside of `__tests__` folders, as well as any files with a suffix of `.test` or `.spec` (e.g. `Component.test.js` or `Component.spec.js`). It will also find files called `test.js` or `spec.js`. See also [`testMatch` [array&lt;string&gt;]](#testmatch-arraystring), but note that you cannot specify both options.
+The pattern or patterns Jest uses to detect test files. See also [`testMatch` [array&lt;string&gt;]](#testmatch-arraystring), but note that you cannot specify both options.
 
-The following is a visualization of the default regex:
+Example:
 
-```bash
-├── __tests__
-│   └── component.spec.js # test
-│   └── anything # test
-├── package.json # not test
-├── foo.test.js # test
-├── bar.spec.jsx # test
-└── component.js # not test
+```json
```

Comment:
It would be better to split documentation change to a separate PR.

First, `next` version docs provide configuration examples in JS and TS (not JSON). See: 

https://github.com/facebook/jest/blob/6e5b1d60a1214e792b5229993b5475445e9c1a6e/docs/Configuration.md?plain=1#L2078-L2095

Second, if applicable the change should be made in [all versioned docs](https://github.com/facebook/jest/tree/main/website/versioned_docs).

All that would simply make this PR too noisy (;